Modules/FT0/include/FT0/DigitQcTask.h

```diff
@@ -107,6 +109,37 @@ class DigitQcTask final : public TaskInterface
   std::unique_ptr<TH1F> mHistNumADC;
   std::unique_ptr<TH1F> mHistNumCFD;
 
+  std::map<int, bool> mMapTrgSoftware;
+  enum TrgModeSide { kAplusC,
+                     kAandC,
+                     kA,
+                     kC
+  };
+  enum TrgModeThresholdVar { kAmpl,
+                             kNchannels
+  };
+  enum TrgComparisonResult { kSWonly,
+                             kTCMonly,
+                             kNone,
+                             kBoth
+  };
+  const int mNChannelsA = 96;
```

Comment:
do we have it defined somewhere already?

Reply:
https://github.com/AliceO2Group/AliceO2/blob/6061027f33dbf3332e41b5e5ef5f72373bf93d7f/Detectors/FIT/FT0/base/include/FT0Base/Geometry.h#L52-L53
Also there are should be limitation for C side for ref channel suppressing in case of vertex calculation.

Reply:
@afurs should we put it into `FT0Base/Geometry.h`, or to consul config? I think config would be better because channel map can change, so we should just have a list of channels in consul that corresponds to reference PMT channels, and most QC tasks should ignore them completely

Reply:
constant replace with the ones from Geometry.h and limitation for C added